src/server/routes/collection.js

```diff
@@ -280,4 +285,31 @@ router.post('/:collectionId/add', auth.isAuthenticated, auth.isCollectionOwnerOr
 	}
 });
 
+router.post('/:collectionId/collaborator/remove/:collaboratorId', auth.isAuthenticated, async (req, res, next) => {
+	try {
+		const {collection} = res.locals;
+		const collaboratorId = parseInt(req.params.collaboratorId, 10);
+		if (parseInt(req.user.id, 10) !== collaboratorId ||
+			!collection.collaborators.find(collaborator => collaborator.id === collaboratorId)) {
```

Comment:
If I read this correctly, I won't be able to call this endpoint if I'm the owner of the collection, which would be unexpected.